src/pages/console/formbuilder/overrides.mdx

```diff
@@ -29,4 +26,6 @@ function ProductFormContainer({ productType }) {
 }
 ```
 
-**Note:** we do not recommend overriding properties that are already set by the generated form. This could lead to unexpected behavior during runtime. Verify the set properties by navigating to the component in the `src/ui-components/<your-form-component>.jsx` file.
+The `productType` is dynamically passed into the function and utilizes container component as a React prop (e.g. `shoe`, `phone`).
```

Comment:
We allow arbitrary props to be passed in?

Reply:
removed this line as it does not add value to the example